contribs/gmf/less/map.less

```diff
@@ -22,6 +29,20 @@
   }
 }
 
+/**
+ * Only affects webkit desktop browsers
+ */
+@media screen and (-webkit-min-device-pixel-ratio:0) {
```

Comment:
yes, it adds the light blue shadow around the buttons. it is the default behaviour of webkit browsers. i guess we want to keep it?